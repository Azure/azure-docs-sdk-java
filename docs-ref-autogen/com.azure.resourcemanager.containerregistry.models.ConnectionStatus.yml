### YamlMime:JavaType
uid: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus"
fullName: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus"
name: "ConnectionStatus"
nameWithType: "ConnectionStatus"
summary: "Defines values for Connection<wbr>Status."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedClassMethods:
- classRef: "<xref href=\"com.azure.core.util.ExpandableStringEnum?alt=com.azure.core.util.ExpandableStringEnum&text=ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
  methodsRef:
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)?alt=com.azure.core.util.ExpandableStringEnum.<T>fromString&text=<T>fromString\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)?alt=com.azure.core.util.ExpandableStringEnum.<T>values&text=<T>values\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)?alt=com.azure.core.util.ExpandableStringEnum.equals&text=equals\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.hashCode()?alt=com.azure.core.util.ExpandableStringEnum.hashCode&text=hashCode\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.util.ExpandableStringEnum.toString()?alt=com.azure.core.util.ExpandableStringEnum.toString&text=toString\" data-throw-if-not-resolved=\"False\" />"
- classRef: "java.lang.<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html\">Object</a>"
  methodsRef:
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#clone--\">clone</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#finalize--\">finalize</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#getClass--\">getClass</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notify--\">notify</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#notifyAll--\">notifyAll</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait--\">wait</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait-long-\">wait</a>"
  - "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html#wait-long-int-\">wait</a>"
syntax: "public final class ConnectionStatus extends ExpandableStringEnum<ConnectionStatus>"
constructors:
- uid: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.ConnectionStatus()"
  fullName: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.ConnectionStatus()"
  name: "ConnectionStatus()"
  nameWithType: "ConnectionStatus.ConnectionStatus()"
  syntax: "public ConnectionStatus()"
fields:
- uid: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "ConnectionStatus.APPROVED"
  summary: "Static value Approved for Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.containerregistry.models.ConnectionStatus?alt=com.azure.resourcemanager.containerregistry.models.ConnectionStatus&text=ConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final ConnectionStatus APPROVED"
  desc: "Static value Approved for ConnectionStatus."
- uid: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.DISCONNECTED"
  fullName: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.DISCONNECTED"
  name: "DISCONNECTED"
  nameWithType: "ConnectionStatus.DISCONNECTED"
  summary: "Static value Disconnected for Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.containerregistry.models.ConnectionStatus?alt=com.azure.resourcemanager.containerregistry.models.ConnectionStatus&text=ConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final ConnectionStatus DISCONNECTED"
  desc: "Static value Disconnected for ConnectionStatus."
- uid: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.PENDING"
  fullName: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.PENDING"
  name: "PENDING"
  nameWithType: "ConnectionStatus.PENDING"
  summary: "Static value Pending for Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.containerregistry.models.ConnectionStatus?alt=com.azure.resourcemanager.containerregistry.models.ConnectionStatus&text=ConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final ConnectionStatus PENDING"
  desc: "Static value Pending for ConnectionStatus."
- uid: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.REJECTED"
  fullName: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.REJECTED"
  name: "REJECTED"
  nameWithType: "ConnectionStatus.REJECTED"
  summary: "Static value Rejected for Connection<wbr>Status."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.containerregistry.models.ConnectionStatus?alt=com.azure.resourcemanager.containerregistry.models.ConnectionStatus&text=ConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final ConnectionStatus REJECTED"
  desc: "Static value Rejected for ConnectionStatus."
methods:
- uid: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.fromString(java.lang.String)"
  fullName: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.fromString(String name)"
  name: "fromString(String name)"
  nameWithType: "ConnectionStatus.fromString(String name)"
  summary: "Creates or finds a Connection<wbr>Status from its string representation."
  modifiers:
  - "static"
  parameters:
  - description: "a name to look for."
    name: "name"
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/lang/String.html\">String</a>"
  syntax: "public static ConnectionStatus fromString(String name)"
  desc: "Creates or finds a ConnectionStatus from its string representation."
  returns:
    description: "the corresponding ConnectionStatus."
    type: "<xref href=\"com.azure.resourcemanager.containerregistry.models.ConnectionStatus?alt=com.azure.resourcemanager.containerregistry.models.ConnectionStatus&text=ConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
- uid: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.values()"
  fullName: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.values()"
  name: "values()"
  nameWithType: "ConnectionStatus.values()"
  modifiers:
  - "static"
  syntax: "public static Collection<ConnectionStatus> values()"
  returns:
    description: "known ConnectionStatus values."
    type: "<a href=\"https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html\">Collection</a>&lt;<xref href=\"com.azure.resourcemanager.containerregistry.models.ConnectionStatus?alt=com.azure.resourcemanager.containerregistry.models.ConnectionStatus&text=ConnectionStatus\" data-throw-if-not-resolved=\"False\" />&gt;"
type: "class"
desc: "Defines values for ConnectionStatus."
metadata: {}
package: "com.azure.resourcemanager.containerregistry.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-containerregistry:2.19.0
