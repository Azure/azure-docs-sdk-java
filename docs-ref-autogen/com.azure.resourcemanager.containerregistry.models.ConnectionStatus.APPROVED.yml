### YamlMime:JavaMember
uid: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "ConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.containerregistry.models.ConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "ConnectionStatus.APPROVED"
  summary: "Static value Approved for ConnectionStatus."
  syntax: "public static final ConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.containerregistry.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-containerregistry:2.13.0
